ironfish/src/wallet/wallet.ts

```diff
@@ -690,6 +690,10 @@ export class Wallet {
       },
     )
 
+    if (options.rawTransaction === true) {
+      return raw
+    }
```

Comment:
I was expecting that `mint` and `burn` would follow the same logical flow as `send` with create, post, and broadcast as separate steps and separate RPC calls.

Should we do mints differently?